compiler/main/driver.cpp

```diff
@@ -1584,6 +1597,35 @@ static void checkUnsupportedConfigs(void) {
   }
 }
 
+static void checkRuntimeBuilt(void) {
+  // no need for a runtime to be built for chpldoc
+  if (fDocs) {
+    return;
+  }
+
+  std::string runtime_dir(CHPL_RUNTIME_LIB);
+  runtime_dir += "/";
+  runtime_dir += CHPL_RUNTIME_SUBDIR;
+
+  if (!isDirectory(runtime_dir.c_str())) {
+    const char* module_home = getenv("CHPL_MODULE_HOME");
+    if (module_home) {
+      USR_FATAL("The requested configuration is not included in the module. "
```

Comment:
Is there a need to keep the original error in runtime/etc/Makefile.include still?

Reply:
I don't think so, but I didn't see it as harming anything there. Would you like me to remove it?